src/parachain/issue.ts

```diff
@@ -41,12 +52,13 @@ export interface IssueAPI extends TransactionAPI {
      * parachain (incurring an extra request).
      * @returns An object of type {singleVault, maxTotal, vaultsCache}
      */
-    getRequestLimits(vaults?: Map<AccountId, MonetaryAmount<WrappedCurrency, BitcoinUnit>>): Promise<IssueLimits>;
+    getRequestLimits(vaults?: Map<string, MonetaryAmount<WrappedCurrency, BitcoinUnit>>): Promise<IssueLimits>;
```

Comment:
I was thinking that `InterbtcPrimitivesVaultId` would be tricky to construct from the UI, but I think it will actually be fine. I'll use the strong type